src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -137,11 +140,13 @@
             [:div {:style {:marginBottom "0.25em"}}
              [buttons/Button {:icon :add-new :text "Add new"
                               :onClick (fn [_]
-                                         (swap! state update :attributes conj ["" ""])
+                                         (swap! state #(-> %
+                                                           (assoc :edit-attribute-table-key (edit-attribute-table-key))
+                                                           (update :attributes conj ["" ""])))
                                          ;; have to do this by ID not ref, since the fields are generated within Table
                                          (after-update #(.focus (.getElementById js/document "focus"))))}]])
           [Table
-           {:key (str editing? (count (:attributes @state)))
+           {:key (or (:edit-attribute-table-key @state) (edit-attribute-table-key))
```

Comment:
This still worries me. If there's a `props` change before the first time that the key gets stored in state, then it will force a re-render of the entire table. Better to implement `:get-initial-state` and set the initial key there.